app/storage.js

```diff
@@ -37,8 +37,15 @@ const getSuppressedReport = async () => {
   return blob.download()
 }
 
+const getCPATSummary = async () => {
+  containersInitialised ?? await initialiseContainers()
+  const blob = await reportContainer.getBlockBlobClient(config.summaryReportName)
+  return blob.download()
```

Comment:
Do wonder if this is the most performant way.  Would need to dig into the blob storage to remind myself but feels like this downloads the full doc first, then offers as a download.  Rather than streaming the full thing.

Not suggesting changing as part of this as this is consistent with the others, but probably worth a ticket to investigate further to avoid future performance bottlenecks.

Reply:
Ticket raised for this: https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/316811